client/src/www/app/fake_tunnel.ts

```diff
@@ -15,6 +15,8 @@
 import {Tunnel, TunnelStatus, ShadowsocksSessionConfig} from './tunnel';
 import * as errors from '../model/errors';
 
+export const FakeBrokenHostName = '192.0.2.1';
+export const FakeUnreachableHostName = '10.0.0.24';
```

Comment:
nit: Prefer `CONSTANT_CASE` for these.

https://google.github.io/styleguide/tsguide.html#identifiers-constants